pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -67,3 +67,15 @@ def set_standard_output_variables(self):
                 "r_p [m]": var.r_p * param.R_p,
             }
         )
+
+    def set_reactions(self):
+
+        # Should probably refactor as this is a bit clunky at the moment
+        # Maybe each reaction as a Reaction class so we can just list names of classes
+        icd = " interfacial current density"
+        self.reactions = {
+            "main": {
+                "Negative": {"s": 1, "aj": "Negative electrode" + icd},
```

Comment:
I can never remember what `s` is. Should it be (1-t+)?

Reply:
well spotted